src/open_inwoner/templates/pages/cases/status_outer.html

```diff
@@ -9,7 +9,7 @@
     <div class="cases__detail" id="cases-detail-content"><div>
 
     <div class="loader-container" id="spinner-container">
-        <div class="spinner" id="spinner-id" hx-get="{{ hxget }}" hx-trigger="load" hx-target="#cases-detail-content">
+        <div class="spinner" id="spinner-id" hx-get="{{ hxget }}" hx-trigger="load" hx-target="#cases-detail-content" hx-include="[name='page']">
```

Comment:
Out of curiosity: is this AJAX specific?

Reply:
No longer needed if we don't use pagination